specs/Pills.md

```diff
@@ -0,0 +1,188 @@
+# [Pills]()
+
+## Background
+
+PIlls should be used when representing an input, as a way to filter content, or to represent an attribute.
+
+## Prior Art
```

Comment:
Should this just be a link to Open UI research ?

Reply:
The research hasn't been merged yet 😔

Reply:
just link the PR I guess :)